src/ontology/mondo.Makefile

```diff
@@ -1298,6 +1302,24 @@ update-subclass-sync: $(TMPDIR)/subclass-confirmed.robot.owl tmp/subclass-axioms
 		make NORM
 		mv NORM $(SRC)
 
+tmp/synonyms-axioms.owl: $(SRC)
+	$(ROBOT) filter --input $(SRC) --term oboInOwl:hasExactSynonym --axioms annotation --preserve-structure false --trim false \
+		--drop-axiom-annotations "oboInOwl:source=~'(DOID|ICD10CM|icd11.foundation|NCIT|OMIM|OMIMPS|Orphanet):.*'" \
+		-o $@
```

Comment:
why does `drop-axiom-annotations` not include all sources in [sync-synonyms.confirmed.robot.tsv](https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/refs/heads/sync1-synonyms/src/ontology/reports/sync-synonym/sync-synonyms.confirmed.robot.tsv)

what about confirmed narrow and related synonyms?

Reply:
> why does drop-axiom-annotations not include all sources in [sync-synonyms.confirmed.robot.tsv](https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/refs/heads/sync1-synonyms/src/ontology/reports/sync-synonym/sync-synonyms.confirmed.robot.tsv)

Did I miss any apart from ICD10WHO? it was just an oversight, my bad.. All should be there now.

> what about confirmed narrow, related, or broad synonyms?

All fixed now, thank you!

Note there are a NO broad synonyms coming in from the sync.. Maybe check if its true that there should be zero broad synonyms in the syn.

Reply:
Can any of these ever end up in the synonym sync? Theoretically?

Reply:
I'll say with 95+% certainty that no. I think the icd11 ones get filtered out in the signature step. `ICD10WHO2010` I've never seen.

Reply:
Since the synonym sync uses the sssom file and then only works with the sources there are components for in mondo-ingest it's limited to that.